Makefile

```diff
@@ -27,6 +27,10 @@ SDK_ALL_PKGS=${SDK_COMPA_PKGS} ${SDK_EXAMPLES_PKGS}
 RUN_NONE=-run '^$$'
 RUN_INTEG=-run '^TestInteg_'
 
+CODEGEN_RESOURCES_PATH=$(shell pwd)/codegen/smithy-aws-go-codegen/src/main/resources/software/amazon/smithy/aws/go/codegen
+ENDPOINTS_JSON=${CODEGEN_RESOURCES_PATH}/endpoints.json
+ENDPOINT_PREFIX_JSON=${CODEGEN_RESOURCES_PATH}/endpoint-prefix.json
+
 LICENSE_FILE=$(shell pwd)/LICENSE.txt
 
 all: generate unit
```

Comment:
Should `all` call sync-endpoint-models?

Reply:
I wasn't sure, how we wanted to do that since this really is a prep for releasing a update. I err'ed on the side of caution to not make it get the v1 SDK when uneeded. If the SDK had a similar location it could sync the Smithy API models from I'd vote it be included with that set of tasks.